src/lib/components/ui/select/select.svelte

```diff
@@ -5,18 +5,23 @@
 	import { withSelectStore } from './select-store.svelte'
 
 	type Dimension = 'default' | 'large' | 'compact' | 'small'
-	interface Props extends HTMLInputAttributes {
+	interface Props extends Omit<HTMLInputAttributes, 'onchange' | 'oninput'> {
```

Comment:
I am not sure the `onchange` and `oninput` is really needed if the value can be bound. I am also not a fan of changing the semantics of the existing functions.

For now I would just remove them. I was thinking about use-cases when you may want more customised UX, but they are more for text inputs. For those the best solution I found is to be able to provide a custom input element as default slot. For example: https://flowbite-svelte.com/docs/forms/input-field#Advanced_usage

Reply:
Fair point, the `oninput` is definitely extra, `onchange` can be reason about. I removed them for now, but that means that these events are never triggered. 

We likely need a more thorough discussion what in general will be the in/out interfaces for inputs to be consistent.